bin/pycbc_upload_xml_to_gracedb

```diff
@@ -153,22 +198,44 @@ for event in coinc_table:
     xmldoc.childNodes[-1].appendChild(coinc_inspiral_table_curr)
     xmldoc.childNodes[-1].appendChild(coinc_event_map_table_curr)
     xmldoc.childNodes[-1].appendChild(sngl_inspiral_table_curr)
-    ligolw_utils.write_filename(xmldoc, "tmp_coinc_xml_file.xml")
+
+    if args.filename_T050017:
+        ifos = sorted([sngl.ifo for sngl in sngl_inspiral_table_curr])
+        ifos_str = ''.join(ifos)
+        id_str = args.filename_T050017
+        filename_xml = "{}-PyCBC_{}-{:d}-1.xml".format(ifos_str, id_str, time)
+        filename_psd = "{}-PyCBC_{}-{:d}-1_psd.xml.gz".format(ifos_str, id_str, time)
+    else:
+        print("Using default filenames")
```

Comment:
I am not sure this logging is necessary given that you do not have a corresponding output in the other branch of the `if`, but if you decide to keep it, replace `print()` with `logging.info()`.

Reply:
I've removed the option to have default filenames now, so this has been removed